metadata/stream/rationale.yaml

```
@@ -135,10 +133,14 @@ architectures:
         # change.
         image: Fedora:CoreOS:stable:latest
       gcp:
-        # We could give a specific image name here, but we probably want
-        # users to always use an image family.  So this is a static string,
-        # and represents advice rather than a value we might change.
-        image: projects/fedora-cloud/global/images/family/fedora-coreos-stable
+        # Ideally users use the project + family.  These are static strings,
+        # and represent advice rather than a value we might change.
+        project: fedora-coreos-cloud
+        family: fedora-coreos-stable
+        # As an alternative, we also list the currently recommended image
+        # and its release.
+        release: 35.20220313.3.1
```

```suggestion
        release: 30.1.2.3
```

All other instances of `release` in this file are `30.1.2.3`.